pkg/pipeline/sink/segments.go

```diff
@@ -139,11 +153,7 @@ func (s *SegmentSink) handleClosedSegment(update SegmentUpdate) {
 	go func() {
 		defer close(update.uploadComplete)
 
-		_, size, err := s.Upload(segmentLocalPath, segmentStoragePath, s.outputType, true)
-		if err != nil {
-			s.callbacks.OnError(err)
-			return
-		}
+		_, size, _ := s.Upload(segmentLocalPath, segmentStoragePath, s.outputType, true, "segment")
```

Comment:
I missed this one earlier, in the case of bad credentials the egress should fail

Reply:
oh... ok.  I'll add
```
if err != nil {
			s.callbacks.OnError(err)
			return
		}
```